green_cli/green.py

```diff
@@ -495,6 +495,15 @@ def getreceiveaddress(session, details):
     """Get a new receive address"""
     return gdk.get_receive_address(session.session_obj, json.dumps(details))
 
+@green.command()
+@click.argument('subaccount', type=int)
+@click.argument('last_pointer', type=int)
+@with_login
+@print_result
+def getpreviousaddresses(session, subaccount, last_pointer):
+    """Get previously generated addresses"""
+    return json.loads(gdk.get_previous_addresses(session.session_obj, subaccount, last_pointer))
```

Comment:
Can be written:

`return session.get_previous_addresses(subaccount, last_pointer)`

Reply:
Changed to a Call interface for liquid now.